lib/webdrivers/chromedriver.rb

```diff
@@ -16,7 +16,13 @@ def current_version
       end
 
       def latest_version
-        raise StandardError, 'Can not reach site' unless site_available?
+        unless site_available?
```

Comment:
Just a thought, if we could add a configuration option to disable all network calls, then we could make an early `return current_version` before this line, if the option is set.

Reply:
Yeah, I'm planning on doing that very soon.